app/assets/js/scripts.js

```diff
@@ -182,6 +176,23 @@ app.hookupSteps = function() {
 
 };
 
+app.setPublisher = function($publisher) {
+  app.state.publisher_id = $publisher.data('publisher-id');
+
+  // todo: make eventsArePolygons a property of a publisher
+  app.eventsArePolygons = $publisher.data('publisher-title').match(/Leaf Collection/);
```

Comment:
Is there a way to make this matching less "magical"?  Leaf Collection might not be the only polygon point of interest.  Is this something that could/should be moved to the Publisher model?